crates/shared/src/http_solver_api.rs

```diff
@@ -87,6 +87,7 @@ impl HttpSolverApi {
         }
         let body = serde_json::to_string(&model).context("failed to encode body")?;
         tracing::trace!("request {}", body);
+        println!("{}", body);
```

Comment:
nit: Rogue print statement.
```suggestion
        println!("{}", body);
```